### DISCUSSION TITLE
Add data to Table that already has some information

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#13224] [Help]
Hey,

I want to split the data in 1 row into 2 screens.
That is, when entering information A in screen 1, press save, then go to find information A that was recorded in screen 2 and add the remaining information.
Database is Budibase DB

Please help.

**Comment by ConorWebb96** [comment]
Hey @LTCTHITA,

To ensure smooth transition from screen 1 to screen 2 while retaining the data created in the initial form, consider the following approach:

1. **Passing Data**: Upon saving the form in screen 1, pass the _id of the saved row to screen 2. This can be achieved by using the state action and providing it with the `saved row._id` value.
2. **Screen 2 Setup**: Configure screen 2 to utilise a form block with its schema matching that of the form used in screen 1.
3. **Retrieving Data**: Utilise the state variable holding the _id value to retrieve the corresponding record from the database. Place this _id value in the designated field within the form block, labeled "Row ID."
4. By following these steps, screen 2 will seamlessly retrieve the previously saved data, allowing you to continue editing from where you left off.

I hope this clarifies the process for you. Let me know if you need further assistance.

**Comment by LTCTHITA** [comment]
Hey @ConorWebb96 ,

I don't understand how it must be done. 

The required steps are
1. Fill in information in Screen1
2. Data recorded in screen1 is displayed in screen2
3. Fill in information in Screen2
4. Data recorded in screen2 goes to the same row as data recorded in screen1

![image](https://github.com/Budibase/budibase/assets/93240913/5dc4eb13-3353-4764-aec7-638e1d166dbf)

![image](https://github.com/Budibase/budibase/assets/93240913/05abbf5a-80e7-4a88-8bd8-97ab6e012695)
